monkestation/code/modules/antagonists/borers/code/focus_datum.dm

```diff
@@ -0,0 +1,77 @@
+/datum/borer_focus
```

Comment:
Wouldn't it perhaps be better to just add `gain_text` and `lose_text` vars rather than manually calling to_chat in every subtype?